pkg/ctl/delete/cluster.go

```
@@ -193,8 +193,14 @@ func deleteFargateProfiles(cmd *cmdutils.Cmd, ctl *eks.ClusterProvider) error {
 	)
 	profileNames, err := awsClient.ListProfiles()
 	if err != nil {
+		if fargate.IsUnauthorizedError(err) {
+			logger.Debug("Fargate: unauthorized error: %v", err)
+			logger.Info("Account is not authorized to use Fargate. Ignoring error")
```

Should we log the error even if we are ignoring it?

---

The error is logged at debug level above this line. Are you suggesting adding it to info level also?

---

yeah, like info: "account is not auth.... Ignoring error (NotAuthorized:...)"

---

Also I guess it should be `account...` without the capital A

---

```suggestion
			logger.Info("account is not authorized to use Fargate. Ignoring error")
```